### DISCUSSION TITLE
Ability to add a link to the shopping cart

### DISCUSSION THREAD

**Discussion by Brentacon** [gameserverapp/Platform#867] [Suggestions]
Hi, So the shopping cart doesnt work for crossplay, so I was wondering if we could add a link to the shop, so when ppl press the cart it opens an external link, this would mean it is not an ingame shop but just a link, is this possible? it would make life easier for people trying to get to the app website. best regards.

**Comment by mvaessen** [comment]
Hi, check out the vote screen feature. It lets you add buttons https://docs.gameserverapp.com/dashboard/integration_mod#vote-screen

**Reply by Brentacon** [reply]
Thank you Max I have vote buttons up but did not even think to use them for shop aswell haha, legend
